server/custom_vms.go

```diff
@@ -487,45 +515,116 @@ func (lc *localNetwork) restartNodesWithWhitelistedSubnets(
 
 func addSubnetValidators(
 	ctx context.Context,
-	subnetIDs []ids.ID,
+	nodeInfos map[string]*rpcpb.NodeInfo,
+	platformCli platformvm.Client,
 	baseWallet *refreshableWallet,
-	validatorIDs []ids.NodeID,
+	subnetIDs []ids.ID,
 ) error {
-	println()
-	color.Outf("{{green}}adding all nodes as subnet validator for each subnet{{/}}\n")
+	fmt.Println()
+	color.Outf("{{green}}adding the nodes as subnet validators{{/}}\n")
```

Comment:
Can we include which nodes we're referring to in this log?

Reply:
Later on in the function we do log the added nodes. I shall avoid making an iteration to get the list of nodes to be then included in this log you refer to.